README.adoc

```diff
@@ -163,13 +165,13 @@ include::finish/system/src/main/liberty/config/server.xml[]
 
 
 // =================================================================================================
-== Creating configuration profile in properties level
+== Create the configuration profile at the property level
 
-The config profile feature in MicroProfile Config allows configurations for different environments such as development, testing, and production while only a single profile is active. The active profile is specified by using the `mp.config.profile` property, which can be set in any of the https://download.eclipse.org/microprofile/microprofile-config-3.0/microprofile-config-spec-3.0.html#configsource[ConfigSources^] or at application startup. Once it is set, the corresponding set of configuration properties that are associated with the active profile are used. 
+The config profile feature in MicroProfile Config supplies configurations for different environments such as development, testing, and production while only a single profile is active. The active profile is specified by using the `mp.config.profile` property, which can be set in any of the https://openliberty.io/guides/microprofile-config-intro.html#background-concepts[ConfigSources^] or at application startup. Once it is set, the corresponding set of configuration properties that are associated with the active profile are used. 
```

Comment:
A concern to use interactive guide for the reference of the ConfigSources because interactive guides will be deprecated. If we do not have a better link, we can keep to use the interactive guide until it really be deprecated.

Reply:
another possibility:
https://openliberty.io/docs/latest/external-configuration.html#_default_configuration_sources

We should avoid pointing them to the spec if possible

Reply:
That link works

Reply:
updated